.travis.yml

```diff
@@ -74,6 +74,8 @@ script:
     travis_retry make test-ios | xcpretty;;
   Mac)
     travis_retry make test-macos | xcpretty;;
+  SPM)
```

Comment:
Are changes to (or even the existence of) `.travis.yml` still necessary since using CirrusCI?